.github/azure-pipelines.yml

```diff
@@ -96,9 +103,16 @@ jobs:
   - template: jobs/ios.yml
     parameters:
       name: 'iOS_Xcode152'
-      vmImage: 'macOS-13'
+      vmImage: 'macOS-14'
       xCodeVersion: 15.2
-      simulator: 'iPhone 14'
+      simulator: 'iPhone 15'
```

Comment:
Why bump the simulator? is this the new minimum hardware/OS combo for BabylonNative moving forward?

Reply:
`macOS-14` image doesn't include `iPhone 14` anymore.

https://github.com/actions/runner-images/blob/macos-14/20250928.1654/images/macos/macos-14-Readme.md